cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Poll.hs

```diff
@@ -0,0 +1,74 @@
+module Cardano.CLI.EraBased.Options.Governance.Poll
+  ( pGovernancePollCmds,
+  )
+where
+
+import           Cardano.Api
+
+import           Cardano.CLI.EraBased.Commands.Governance.Poll (GovernancePollCmds (..))
+import           Cardano.CLI.EraBased.Options.Common
+import           Cardano.Prelude (catMaybes, isInfixOf)
+
+import           Control.Monad (when)
+import           Data.Foldable
+import           Options.Applicative hiding (help, str)
+import qualified Options.Applicative as Opt
+
+pGovernancePollCmds ::
+  () =>
+  CardanoEra era ->
+  Maybe (Parser (GovernancePollCmds era))
```

Comment:
Can we have the arrows on the LHS on the next line?

Reply:
:+1: Did the change:

```diff
-pGovernancePollCmds ::
-  () =>
-  CardanoEra era ->
-  Maybe (Parser (GovernancePollCmds era))
+pGovernancePollCmds :: ()
+  => CardanoEra era
+  -> Maybe (Parser (GovernancePollCmds era))
```